example/react-native.config.js

```diff
@@ -26,7 +26,30 @@ module.exports = {
       ),
     },
     ios: {
-      project: "ios/ReactTestApp-Dummy.xcodeproj",
+      project: (() => {
```

Comment:
This needs to be ported to `scripts/configure.js` as well:
https://github.com/microsoft/react-native-test-app/blob/b72bb100d938d417fe7c0228693bcea29b84f37a/scripts/configure.js#L244-L246